source/use-cases/sensitive-data-encryption.txt

```diff
@@ -57,12 +57,222 @@ cubilia Curae; Nunc non interdum purus, ultricies laoreet tortor.
 B. Define a JSON Schema
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-Proin non mi a felis luctus vulputate nec pharetra odio. Ut pretium
-scelerisque nulla, sed euismod nisi pellentesque sit amet. Proin mollis
-mauris eu libero ullamcorper rutrum. Pellentesque vitae enim sed magna
-accumsan maximus et quis mi. Ut vel laoreet turpis. Vivamus vitae odio
-sed arcu finibus pretium vel maximus nisi. Nullam gravida consequat
-porta.
+MongoDB uses JSON Schema to validate data that is written to a collection.
+Within the context of Field Level Encryption, JSONSchema is used to
+specify several configuration details.
+For instance, you can specify the form of encryption you want to use  -
+deterministic or random.
+You can also specify the BSON Type of the field you are encrypting. In
+addition, you can 
+specify the key you want to use. This is particularly important, because
+it means that you can use different keys for different fields.
+
+The following example goes over the creation of a schema for a collection
+that represents patient data
+in a medical office. Some fields use deterministic encryption, and others
+use random (probabilistic) encryption. 
+
+The structure for the schema will be as follows:
+
+.. code::
+
+    fullName: String
+    address: Encrypted String using random encryption
+    ssn: Encrypted Int using deterministic encryption
+    last4SSN: Int
+    patientInfo: Object
+    patientInfo.phone Encrypted Int using deterministic encryption
+    patientInfo.provider String
+    medicalRecords: Encrypted array of documents using random encryption
```

Comment:
Can we make it a bit clearer that these bucket into three types of fields:

- Unencrypted fields
- Randomly encrypted fields
- Deterministically encrypted fields

The type information isn't all too helpful except to discuss why we must use random on array fields and we can cover it in the **Problem** section (see my comment below)

Reply:
Done